src/site/shared/containers/ItemNav/index.tsx

```diff
@@ -15,18 +15,20 @@ import {DeleteHandler} from "core/tools/handlers/DeleteHandler";
 import {Component} from "core/models";
 
 import {useSharedDispatch, useSharedSelector} from "shared/utils/hooks/useShared";
-import {useWindowKeyDownEvent} from "shared/utils/hooks/useKeyDownEvent";
+import {useKeyDownEvent, useWindowKeyDownEvent} from "shared/utils/hooks/useKeyDownEvent";
+import {useKey} from "shared/utils/hooks/useKey";
 import {useMousePos} from "shared/utils/hooks/useMousePos";
 import {useDocEvent} from "shared/utils/hooks/useDocEvent";
 import {useHistory} from "shared/utils/hooks/useHistory";
 import {useWindowSize} from "shared/utils/hooks/useWindowSize";
 
-import {OpenItemNav, CloseItemNav, CloseHistoryBox, OpenHistoryBox} from "shared/state/ItemNav";
+import {OpenItemNav, CloseItemNav, CloseHistoryBox, OpenHistoryBox, itemNavReducer} from "shared/state/ItemNav";
 
 import {Draggable} from "shared/components/DragDroppable/Draggable";
 import {DragDropHandlers} from "shared/components/DragDroppable/DragDropHandlers";
 
 import "./index.scss";
+import { boolean } from "yargs";
```

Comment:
I'm going to guess this import was accidentally added by your editor and it can probably be safely deleted